Sources/UtiliKit/Container/Transitioning/ContainerViewControllerTransitionCoordinator.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  ContainerViewControllerTransitionCoordinator.swift
+//  UtiliKit-iOS
+//
+//  Created by Will McGinty on 1/22/19.
+//  Copyright © 2019 Bottle Rocket Studios. All rights reserved.
+//
+
+import UIKit
+
+/// The `ContainerViewControllerTransitionCoordinator` is intended to closely mimic the `UIViewControllerTransitionCoordinator` protocol, with the exception that alongside animations must occur in the container of the transition and can not occur outside.
+public protocol ContainerViewControllerTransitionCoordinator: UIViewControllerTransitionCoordinatorContext {
+    
+    @discardableResult
+    func animate(alongsideTransition animation: ((UIViewControllerTransitionCoordinatorContext) -> Void)?,
+                 completion: ((UIViewControllerTransitionCoordinatorContext) -> Void)?) -> Bool
+    
+    func notifyWhenInteractionEnds(_ handler: @escaping (UIViewControllerTransitionCoordinatorContext) -> Void)
+    func notifyWhenInteractionChanges(_ handler: @escaping (UIViewControllerTransitionCoordinatorContext) -> Void)
+}
+
+/// This class is internal to the framework. It's concrete class type is never leaked and is instead exposed simply as an object conforming to `ContainerViewControllerTransitionCoordinator`. It's purpose is to function as a drop in replacement for the transition coordinator objects vended by UIKit for presentations. Because the container utilizes it's own child-based transition environment, UIKit will not create a transition coordinator for it's transitions.
```

Comment:
The word it's is always short for 'it is' (as in it's raining), or in informal speech, for 'it has'. The word its means 'belonging to it'.